common/bcp47/timezone.xml

```diff
@@ -476,6 +477,164 @@ For terms of use, see http://www.unicode.org/copyright.html
             <type name="zajnb" description="Johannesburg, South Africa" alias="Africa/Johannesburg"/>
             <type name="zmlun" description="Lusaka, Zambia" alias="Africa/Lusaka"/>
             <type name="zwhre" description="Harare, Zimbabwe" alias="Africa/Harare"/>
+            <type name="acre" description="Metazone" alias="Acre" since="40"/>
```

Comment:
Optional: Should we add a new XML attribute, like `type="metazone"` ?

Reply:
I thought of that, and can bring it up in Wednesday's meeting. On the other hand, since the metazones will be all and only the 4 letter codes (other than the deprecated gaza) we wouldn't need to.

Reply:
I think the length is sufficient.